views/stats.py

```diff
@@ -0,0 +1,16 @@
+from flask_app import app
+from models import Session, StatsTVL
+
+
+@app.route("/sherlock_stats")
+def sherlock_stats():
+    with Session() as s:
+        stats = StatsTVL.find_all(s)
```

Comment:
Should we maybe set an upper limit of records returned? Like 1 year (365 rows) or something?

Reply:
Yeah, good idea. You can't do much with the chart in the UI for now anyways, so a hard cap makes sense to me.

@Evert0x thoughts on this?